### PR TITLE
Add a parameter to set arch in lcov action

### PR DESCRIPTION
**Pull request by rafaortiz**
Created an optional parameter to specify the architecture in `lcov` action.

### CODE REVIEW

File: lib/fastlane/actions/lcov.rb [hunk 1]
```diff
@@ -33,7 +33,14 @@ def self.available_options
                                        description: "The output directory that coverage data will be stored. If not passed will use coverage_reports as default value",
                                        optional: true,
                                        is_string: true,
-                                       default_value: "coverage_reports")
+                                       default_value: "coverage_reports"),
+
+          FastlaneCore::ConfigItem.new(key: :arch,
+                                       env_name: "FL_LCOV_ARCH",
+                                       description: "The build arch where will search .gcda files. Default is i386",
+                                       optional: true,
```
Comment (KrauseFx) [inline lib/fastlane/actions/lcov.rb]: Please remove the `optional: true`, as this value is not optional :+1:

**Comment by KrauseFx** [comment]
The tests are failing

```
  1) Fastlane Action List Actions provide a complete documenation Valid return values for fastlane action 'lcov'
     Failure/Error: if action.available_options
     RuntimeError:
       Do not let descriptions end with a '.', since it's used for user inputs as well
     # /home/travis/.rvm/gems/ruby-2.0.0-p598/gems/fastlane_core-0.17.1/lib/fastlane_core/configuration/config_item.rb:23:in `initialize'
```

Sorry to nag you, but the description of the option shouldn't end with a `.`.

**Comment by xfreebird** [comment]
@KrauseFx could you please have a look at this PR, it looks ready :)

File: lib/fastlane/actions/lcov.rb [hunk 2]
```diff
@@ -28,13 +28,16 @@ def self.available_options
                                        env_name: "FL_LCOV_SCHEME",
                                        description: "Scheme of the project"),
 
+          FastlaneCore::ConfigItem.new(key: :arch,
+                                       env_name: "FL_LCOV_ARCH",
+                                       description: "The build arch where will search .gcda files"),
```
Comment (KrauseFx) [inline lib/fastlane/actions/lcov.rb]: This should have the

```ruby
default_value: "i386"
```

right?

Comment (xfreebird) [inline lib/fastlane/actions/lcov.rb]: that's right, good catch.
@rafaortiz please see the comment above.

Comment (rafaortiz) [inline lib/fastlane/actions/lcov.rb]: @KrauseFx @xfreebird Why should have a default_value if is not a optional parameter?

Comment (xfreebird) [inline lib/fastlane/actions/lcov.rb]: @rafaortiz it is convenient, less arguments for the action. Also I would set the default value to ```x86_64```.

**Comment by KrauseFx** [comment]
Looks great, thanks :+1:

**Pull request merged**